tests/inc/Analytics/TracksAnalyticsTest.php

```diff
@@ -58,7 +46,6 @@ public function testInitDoesSetTracksIfTrackingIsEnabledViaFilter(): void {
 		TracksAnalytics::init( $env_config_mock );
 
 		$this->assertInstanceOf( MockTracks::class, TracksAnalytics::get_instance() );
-		$this->assertEquals( [ 'activated_plugin', 'deactivated_plugin', 'save_post' ], array_keys( $GLOBALS['__wordpress_actions'] ) );
```

Comment:
This assertion (and the one below) are mostly just testing WordPress core logic. Mocking WP actions and filters robustly for pure unit tests is a bit redundant and complex, and adds little value. Asserting that the class has been instantiated is enough IMO.